src/main/java/org/peergos/APIService.java

```diff
@@ -16,7 +17,7 @@ public class APIService {
     private final BlockService remoteBlocks;
 
     public APIService(Blockstore store, BlockService remoteBlocks) {
-        this.store = store;
+        this.store = new TypeLimitedBlockstore(store, Set.of(Cid.Codec.Raw, Cid.Codec.DagCbor));
```

Comment:
we shouldn't force everyone to use it.

Reply:
maybe put it in the config, and default to not limiting it?

Reply:
done